_pages/channels/homelab.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: channel
+title: Homelab
+channel: homelab
+permalink: /channels/homelab/
+parent: channels
+---
+
+A place for people have 
```

Comment:
This was me typing something, having a brain fart, going on to other things and completely forgetting a had a half-finished sentence somewhere.